contracts/seed/SeedFactory.sol

```diff
@@ -72,17 +72,34 @@ contract SeedFactory is CloneFactory {
         avatar = _newAvatar;
     }
 
+    /**
+      * @dev                          Deploys Seed contract.
+      * @param _admin                 The address of the admin of this contract. Funds contract
+                                      and has permissions to whitelist users, pause and close contract.
+      * @param _seedToken             The address of the token being distributed.
+      * @param _fundingToken          The address of the token being exchanged for seed token.
+      * @param _successMinimumAndCap  Array containing two params:
+                                        - the minimum distribution threshold
+                                        - the highest possible amount to be raised in wei.
+      * @param _price                 Seed to funding token exchange rate.
+      * @param _startTime             Distribution start time in unix timecode.
+      * @param _endTime               Distribution end time in unix timecode.
+      * @param _vestingDuration       Vesting period duration in days.
+      * @param _vestingCliff          Cliff duration in days.
+      * @param _isWhitelisted         Set to true if only whitelisted adresses are allowed to participate.
+      * @param _fee                   Success fee expressed as a % (e.g. 2 = 2% fee)
+    */
```

Comment:
just a reminder that the comments in the factory and seed be the same.  (painful I know, but I think it is necessary)